website/docs/components/form/masked-input/partials/guidelines/guidelines.md

```diff
@@ -68,6 +68,12 @@ For shorter, simpler forms, indicate optional fields instead.
   <F.Label>Label</F.Label>
 </Hds::Form::MaskedInput::Field>
 
+## Copyable masked input 
+
+The `MaskedInput` component lets you show a Copy Button by setting the `hasCopyButton` property to `true` Users can interact with this button to copy the `MaskedInput's` content to their clipboard. Refer to the [copy button](www.helios.com/copy/copy-button) usage guidelines to learn more about copying content. 
+
+![Masked input with a copy button](/assets/components/form/masked-input/masked-input-has-copy-button.png)
+
 ## Error validation
 
-For error validation recommendations, refer to the [Form patterns](/patterns/form-patterns) documentation.
+For error validation recommendations, refer to the [Form patterns](/patterns/form-patterns) documentation
```

Comment:
Not sure why the period was removed here, maybe in error?

Reply:
Hmm, yeah not sure 🧐